Cargo.toml

```diff
@@ -31,6 +32,7 @@ web-time = "1"
 [dev-dependencies]
 bencher = "0.1"
 serde_derive = "1.0"
+bitcode = "0.5"
```

Comment:
This is extra, remove.

Reply:
@PrismaPhonic hi, thank you for your amazing pr! Would you be able to update this pull request (PR) when you have time?